pkg/virt-api/validating-webhook/validating-webhook.go

```diff
@@ -321,6 +321,25 @@ func validateVirtualMachineSpec(field *k8sfield.Path, spec *v1.VirtualMachineSpe
 		return causes
 	}
 
+	// Validate memory size if values are not negative
+	if spec.Domain.Resources.Requests.Memory().Value() < 0 {
+		causes = append(causes, metav1.StatusCause{
+			Type:    metav1.CauseTypeFieldValueInvalid,
+			Message: fmt.Sprintf("%s is specified as negative value", field.Child("domain", "resources", "requests", "memory").String()),
```

Comment:
Let's align with the error messages from k8s:

```
The Pod "nginx" is invalid: spec.containers[0].resources.requests[memory]: Invalid value: "-4Mi": must be greater than or equal to 0
```

The relevant part is: `"-4Mi": must be greater than or equal to 0`.

Reply:
Done